spec/features/user_sees_a_single_book_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe "As a user" do
+  describe "when I visit a book page" do
+    it "he or she sees the books reviews and content" do
+      book = Book.create(title: "Infinite Jest")
+      u1 = User.create(name: "Matt")
+      u2 = User.create(name: "Tolstoy")
+      r1 = Review.create(body: "A stellar read!", user_id: u1.id, book_id: book.id, rating: 5)
+      r2 = Review.create(body: "Post-modern garbage!", user_id: u2.id, book_id: book.id, rating: 1)
```

Comment:
You can also create these associations in your test data like this:
`r1 = Review.create(body: "A stellar read!", user: u1, book: book, rating: 5)`
Both are considered correct, the one posted here will save you some characters.